ui/ozone/platform/starboard/test/surface_factory_starboard_unittest.cc

```diff
@@ -0,0 +1,77 @@
+// Copyright 2024 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "ui/ozone/platform/starboard/surface_factory_starboard.h"
+
+#include "testing/gtest/include/gtest/gtest.h"
+
+namespace ui {
+
+class SurfaceFactoryStarboardTest : public testing::Test {
+ public:
+  SurfaceFactoryStarboardTest() {}
```

Comment:
`= default;` ?

Reply:
Updated this to initialize the class variable in the constructor instead of SetUp.